cluster-autoscaler/core/scale_up.go

```diff
@@ -704,7 +704,8 @@ func executeScaleUp(context *context.AutoscalingContext, clusterStateRegistry *c
 	if err := info.Group.IncreaseSize(increase); err != nil {
 		context.LogRecorder.Eventf(apiv1.EventTypeWarning, "FailedToScaleUpGroup", "Scale-up failed for group %s: %v", info.Group.Id(), err)
 		aerr := errors.ToAutoscalerError(errors.CloudProviderError, err).AddPrefix("failed to increase node group size: ")
-		clusterStateRegistry.RegisterFailedScaleUp(info.Group, metrics.FailedScaleUpReason(string(aerr.Type())), now)
+		backoff := (strings.Contains(aerr.Error(), "Throttling: Rate exceeded"))
```

Comment:
I think this is a good solution for cloud provider because it leaves the backoff decision to cloud provider. Maybe we should consider this and have some plan? @x13n

Reply:
Not sure whether you're referring to the PR as a whole or to my comment here. I think the idea represented by the PR is right (some errors should not result in backoff), but the distinction between backoff and non-backoff cases belongs in each cloud provider separately.